misc/extension_api_validation/4.3-stable.expected

```diff
@@ -243,3 +243,10 @@ GH-97449
 Validate extension JSON: Error: Field 'classes/GraphEdit/methods/connect_node/arguments': size changed value in new API, from 4 to 5.
 
 Added optional argument to connect_node to specify whether the connection should be automatically deleted if invalid. Compatibility method registered.
+
+
+GH-100521
+---------
+Validate extension JSON: Error: Hash changed for 'classes/CodeEdit/methods/add_code_completion_option', from EB1A746E to 611C3D20. This means that the function has changed and no compatibility function was provided.
```

Comment:
Should be solved with a compatibility method instead

Reply:
It already existed - how to write it properly?

Reply:
You haven't committed this code, it's not in the PR

Reply:
Yes - that why I asked - how to add a second compatibility method to `code_edit.compat.inc` ?